pilot/pkg/networking/plugin/authz/authorization.go

```diff
@@ -46,7 +42,7 @@ func (Plugin) OnOutboundListener(in *plugin.InputParams, mutable *networking.Mut
 		return nil
 	}
 
-	buildFilter(in, mutable)
+	buildFilter(in, mutable, false)
```

Comment:
you should create a logger object, set its field to indicate it's pass through filter chain (or normal filter chain), pass it along the way and use it everywhere.

When later using the logger, you can just print the filter chain information, or provide a function to let the caller to decide whether or not it wants to print the error log.